<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtArteryTechMeshSetup</class>
 <widget class="QDialog" name="QtArteryTechMeshSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtArteryTechMeshSetup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="FileNameLabel">
         <property name="text">
          <string>File Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="FileNameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="CoordinateScaleXDirectionLabel">
         <property name="text">
          <string>Coordinate Scale (X-direction):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CoordinateScaleXDirectionLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="CoordinateScaleYDirectionLabel">
         <property name="text">
          <string>Coordinate Scale (Y-direction):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CoordinateScaleYDirectionLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="CoordinateScaleZDirectionLabel">
         <property name="text">
          <string>Coordinate Scale (Z-direction):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CoordinateScaleZDirectionLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="OkPushButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
